source/Clean.js

```diff
@@ -217,7 +219,7 @@ var cleanTree = function cleanTree ( node, config, preserveWS ) {
                 i -= 1;
                 l -= 1;
                 continue;
-            } else if ( !allowedBlock.test( nodeName ) && !isInline( child ) ) {
+            } else if ( !allowedBlock.regexp.test( nodeName ) && !isInline( child ) ) {
```

Comment:
This could just be `config.allowedBlock.test( ...`

Reply:
I was not aware that `cleanTree` receive config as a parameter. Thank you. That's a good point. I'll fix it. 👍